### PR TITLE
Issue #4935 fix indentation check for nested annotations

### PR DESCRIPTION
**Pull request by ArneLimburg**
Issue #4935

This pull-request fixes the problem

### CODE REVIEW

**Comment by rnveach** [comment]
@ArneLimburg If you wish to get this merged, all CI must be green.

**Comment by romani** [comment]
commit message should be: `Issue #4935: ...... `
All CIs should be satisfied(green).

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java [hunk 1]
```diff
@@ -135,7 +134,11 @@ public void checkIndentation(DetailAST firstNode, DetailAST lastNode, int indent
                             getNextNodeLine(firstNodesOnLines, node),
                             true
                         );
-                    checkAnnotationIndentation(atNode, annotationLines, indentLevel);
+                    checkAnnotationIndentation(
+                        atNode,
+                        annotationLines,
+                        getLineStart(atNode),
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java]: What if the parameter `startIndent` isn't -1? Shouldn't you switch between it's value too?
See line 153, https://github.com/checkstyle/checkstyle/pull/4936/files#diff-1ab987049997f8605718e00e9fe92d25R153 .

**Comment by rnveach** [comment]
@ArneLimburg Only your commit must be in this PR, no other commits.

**Comment by romani** [comment]
we released 8.2 version, please rebase all your PRs to our latest master to avoid CI problems

**Comment by romani** [comment]
@ArneLimburg , do you need help ?

do you still want to continue on this PR ?
if not , please let us know, smb else will continue this issue resolving. This active PR blocks issue to be resolved by smb else.

If no reply in next 2 weeks, I will close this PR to let other contributors to continue.

**Comment by romani** [comment]
@ArneLimburg , I marking your PR as abandoned. Looks like we lost connection with you.

To ALL, feel free to continue work or start it from scratch.

**Comment by romani** [comment]
I am closing this PR as author never did a reply to us.

**Pull request closed without merging**